axelrod/moran.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+from collections import Counter
+import random
+
+import numpy as np
+
+from .deterministic_cache import DeterministicCache
+from .match import Match
+from .player import Player
+from .random_ import randrange
+
+
+def fitness_proportionate_selection(scores):
+    """Randomly selects an individual proportionally to score."""
```

Comment:
I wasn't overly clear:

```
Input:

    scores: list an order list of scores for each player

Returns:

    An index of the above list based on the proportional probability of each sore.
```

I realise this is trivial, just trying to future proof things as much as possible.